src/main/resources/swagger.api/schemas/item.json

```diff
@@ -14,6 +14,43 @@
     "barcode": {
       "type": "string",
       "description": "Unique inventory control number for physical resources, used largely for circulation purposes"
+    },
+    "effectiveLocationId": {
+      "type": "string",
+      "description": "Read only current home location for the item."
+    },
+    "status": {
+      "description": "The status of the item",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the status e.g. Available, Checked out, In transit",
+          "type": "string",
+          "enum": [
```

Comment:
I'd remove this enum restriction and use plain string instead, the reason is that from time to time new statuses are added and if we don't sync them up we will get unmarshalling error.

Reply:
Since Pavel is on vacation, I've decided to remove the enum constraint. We can revert it if we decide it required.